protractor/local.config.ts

```diff
@@ -3,7 +3,7 @@ import { reporter } from './helpers/reporter';
 
 export const config: Config = {
   framework: 'jasmine',
-  specs: ['../test/**/*.spec.js'],
+  specs: ['../test/**/locators.spec.js'],
```

Comment:
Don't change this so that all the tests are executed in the pipeline and not only the one you are adding